CHANGELOG.md

```diff
@@ -16,15 +18,19 @@ and this project adheres to
   `createIntegrationLogger()` to allow for setup and execution with alternative,
   non-generated local configuration.
 
-### Fixed
-
 ## 0.11.4 2020-04-24
 
 ### Fixed
 
 - createIntegrationEntity failing on properties with null values that exist in
   the data-model schema
 
+## 0.11.5 2020-04-26
+
+### Added
+
+- Updated `@jupiterone/data-model@0.3.1`
+
```

Comment:
This should probably be moved above 0.11.4, but nbd

Reply:
Yup, noted in opening comment, since I noticed after pushing up tags etc. Thanks for the approve!

Reply:
Ooof idk how I missed the opening comment. Sorry about that